assembl/static2/js/app/components/debate/survey/header.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Grid, Row } from 'react-bootstrap';
+import { Link } from 'react-router';
+import { Translate } from 'react-redux-i18n';
+import { getPhaseName, getCurrentPhaseIdentifier, getIfPhaseCompletedByIdentifier } from '../../../utils/timeline';
+
+class Header extends React.Component {
+  render() {
+    const { title, imgUrl } = this.props;
```

Comment:
It's a path taken from a graphql API and used like this `style={{ backgroundImage: `url(${imgUrl})` }}`